docs/core/runtime-config/garbage-collector.md

```diff
@@ -462,17 +459,11 @@ For more information about some of these settings, see the [Middle ground betwee
 > [!TIP]
 > If you're setting the option in *runtimeconfig.json*, specify a decimal value. If you're setting the option as an environment variable, specify a hexadecimal value. For example, to specify a heap hard limit of 200 mebibytes (MiB), the values would be 209715200 for the JSON file and 0xC800000 or C800000 for the environment variable.
 
-### Heap limit percent
+### Heap hard limit percent
 
-- Specifies the allowable GC heap usage as a percentage of the total physical memory.
-- If [System.GC.HeapHardLimit](#heap-limit) is also set, this setting is ignored.
+- Specifies the heap hard limit as a percentage of the total physical memory. If the process is running in a memory constraint environment, ie, inside a container with a specified memory limit, the total physical memory is the memory limit; otherwise it's what's available on the machine.
```

Comment:
```suggestion
- Specifies the heap hard limit as a percentage of the total physical memory. If the process is running in a memory-constrained environment, that is, inside a container with a specified memory limit, the total physical memory is the memory limit; otherwise it's what's available on the machine.
```